code/game/jobs/job/government_sector.dm

```diff
@@ -7,4 +7,16 @@
 	spawn_positions = 4
 	supervisors = "Jovian naval officials and Sol colonial law"
 	access = list()
-	alt_titles = list("Sub Pilot", "Peace Officer")
\ No newline at end of file
+	alt_titles = list("Sub Pilot", "Peace Officer")
+
+
+	equip(var/mob/living/carbon/human/H)
```

Comment:
What you currently have:
```DM
/datum/job/government
	// things
	equip(var/mob/living/carbon/human/H)
		if(!H)	return 0
		H.equip_to_slot_or_del(new /obj/item/device/radio/headset/headset_sec(H), slot_l_ear)
		H.equip_to_slot_or_del(new /obj/item/clothing/under/rank/petty_officer(H), slot_w_uniform)
		// more things
```
What Zuhayr (and I, and probably everyone else) would like:
```DM
/datum/job/government
	// things
/datum/job/government/equip(var/mob/living/carbon/human/H)
	if(!H)	return 0
	H.equip_to_slot_or_del(new /obj/item/device/radio/headset/headset_sec(H), slot_l_ear)
	H.equip_to_slot_or_del(new /obj/item/clothing/under/rank/petty_officer(H), slot_w_uniform)
	// more things
```